src/js/pages/jobs/JobsTable.js

```diff
@@ -128,6 +133,26 @@ class JobsTable extends React.Component {
       );
     }
 
+    if (schedules && schedules.length !== 0) {
+      let schedule = schedules[0];
+
+      if (schedule.enabled) {
+        scheduleIcon = (
+          <Tooltip
+            wrapperClassName="tooltip-wrapper icon icon-margin-left"
+            content={prettycron.toString(schedule.cron)}
```

Comment:
this actually seems like something that would be nice on it's own column. but that's another issue